### PR TITLE
fix895-ICCProfile-FalseWarning

### PR DESCRIPTION
**Pull request by clanmills**
Discussed in #895

### CODE REVIEW

File: src/futils.cpp [hunk 1]
```diff
@@ -259,19 +259,22 @@ namespace Exiv2 {
         struct {
             std::string name ;
             Protocol    prot ;
+            bool        url  ; // path.size() > name.size()
```
Comment (piponazo) [inline src/futils.cpp]: [suggestion] By renaming this to `isUrl` we would make the code more readable.

**Review by piponazo — DISMISSED** [review]
LGTM! I just made a tiny suggestion, but it is ok if you prefer to leave the code as it is.

**Review by piponazo — APPROVED** [review]

**Comment by D4N** [comment]
This PR should be also applied to master, right?

**Comment by piponazo** [comment]
Right! I can take care about that later.

**Pull request merged**